src/utils/annotation.rs

```diff
@@ -0,0 +1,132 @@
+use tree_sitter::{Parser, Query, QueryCursor};
```

Comment:
given the filename, it should be in plural form, `annotations.rs`. There's no way to tell if we are going to have more annotations in the future.

Reply:
I've renamed it to `annotations.rs`